topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,767 @@
+>    - *"Number of top genes"*: `50`
+{: .hands_on}
+
+{% icon congratulations %}Well done!  I strongly suggest have a play with the **Inspect AnnData** {% icon tool %} on your final `Pre-processed object` to see the wealth of information that has been added. You are now ready to move along to further filtering! There is a cheat that may save you time in the future though...
+
+# 7: Pulling single cell data from public resources
+
+If you happen to be interested in analysing publicly available data, particularly from the [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home), you may be interested in the following tool which rather skips forward all these steps in one! For this tutorial, the dataset can be seen [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/downloads) with experiment id of `E-MTAB-6945`.
+
+> ### {% icon hands_on %} Hands-on: Retrieving data from Single Cell Expression Atlas
+>
+> 1. {% tool [EBI SCXA Data Retrieval](https://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/retrieve_scxa/retrieve_scxa/v0.0.2+galaxy2) %} with the following parameters:
+>    - *"SC-Atlas experiment accession"*: `E-MTAB-6945`
+>    - *"Choose the type of matrix to download"*: `Raw filtered counts`
+>
+> Now we need to transform this into an AnnData objects
+>
+> 2. {% tool [Scanpy Read10x](https://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_read_10x/scanpy_read_10x/1.6.0+galaxy0) %} with the following parameters:
+>    - *"Expression matrix in sparse matrix format (.mtx)"*: `EBI SCXA Data Retrieval on E-MTAB-6945 matrix.mtx (Raw filtered counts)`
+>    - *"Gene table"*:  `EBI SCXA Data Retrieval on E-MTAB-6945 genes.tsv (Raw filtered counts)`
+>    - *"Barcode/cell table"*: `EBI SCXA Data Retrieval on E-MTAB-6945 barcodes.tsv (Raw filtered counts)`
+>    - *"Cell metadata table"*: `EBI SCXA Data Retrieval on E-MTAB-6945 exp_design.tsv`
+{: .hands_on}
+
+It's important to note that this matrix is processed somewhat through the SCXA pipeline, which is quite similar to this tutorial, and it contains any and all metadata provided by their pipeline as well as the authors (for instance, more cell or gene annotations).
+
+# 7. References
+
+ - Moreno P, Huang N, Manning J, et al. User-friendly, scalable tools and workflows for single-cell analysis. bioRxiv; 2020. DOI: 10.1101/2020.04.08.032698.
+ - Lun ATL, Riesenfeld S, Andrews T, et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol. 2019;20(1):63.
+ - Melsted P, Ntranos V, Pachter L. The Barcode, UMI, Set format and BUStools. Bioinformatics. 2019;
+ - Srivastava A, Malik L, Smith T, Sudbery I, Patro R. Alevin efficiently estimates accurate gene abundances from dscRNA-seq data. Genome Biol. 2019;20(1):65.
+ - Zheng GX, Terry JM, Belgrader P, et al. Massively parallel digital transcriptional profiling of single cells. Nat Commun. 2017;8:14049.
```

Comment:
Are these references mentioned in the tutorial at all? If so they could be replaced with the citation feature. If they're just "further reading", hmm. We don't have a built-in solution for that currently but maybe we could figure out something that works nicely using the citation feature (which links the DOIs to the papers making it easier for students to find them.)

Reply:
Done! Hopefully